views/layout.pug

```diff
@@ -0,0 +1,58 @@
+doctype html
+
+html
+	head
+		meta(charset="UTF-8")
+		meta(http-equiv="X-UA-Compatible" content="IE-edge, chrome=1")
+		meta(name="viewport" content="width=device-width, initial-scale=1")
+		title Styleguide Driven Development
+		link(rel="stylesheet", href="/css/core.0.12.0.css")
+		link(rel="stylesheet", href="/css/theme.css")
+	body
+		.svg-sprite(style="position: absolute; width: 0; height: 0;")
```

Comment:
I thought you wanted it below the body tag. Okay, but where is the includes folder? Is there suppose to be one inside the views?